src/System.Device.Gpio/System/Device/Gpio/Drivers/SysFSDriver.Linux.cs

```diff
@@ -0,0 +1,562 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading;
+
+namespace System.Device.Gpio.Drivers
+{
+    /// <summary>
+    /// A GPIO driver for Unix.
+    /// </summary>
+    internal class SysFSDriver : UnixDriver
```

Comment:
Make this public instead, so people can choose to get this behavior instead of libgpiod getting picked

Reply:
Is the casing of this class name (and the class name itself) appropriate? Should it be `SysfsDriver`?

Reply:
great question.. according to kernel.org, `SysfsDriver` might be the right casing for this.